tests/asynctests/combined_asynctests.js

```diff
@@ -0,0 +1,44 @@
+
+function testAll() 
+{
+
+    api.getIntValueAsync( function (val) {
+            compare(val,10);
```

Comment:
This is not run at all, because of the asynchronous nature of the test. You'll need to find a way to wait for the answer, otherwise the tests are useless.